src/components/Plate.vue

```diff
@@ -1,50 +1,51 @@
 <template>
-  <div class="plate">
+  <div class="plate space-y-2">
     <div>
-      <alert ref="alert"></alert>
-      <div class="container-fluid row">
-        <b-modal
-          v-model="exporting"
-          :hide-footer="true"
-          :hide-header="true"
-          :no-close-on-backdrop="true"
-        >
-          <spinner size="huge" message="Exporting..."></spinner>
-        </b-modal>
-        <h3>{{ msg }}: {{ barcode }}</h3>
-        <div class="spacer">
-          <label class="spacer" for="lotNumber">Standards Lot Number:</label>
-          <input id="lotNumber" v-model="lotNumber" type="text" />
-        </div>
-        <div>
-          <button
-            id="save"
-            name="save"
-            class="btn btn-success"
-            @click.prevent="save"
+      <QuanthubMessage ref="alert"></QuanthubMessage>
```

Comment:
Can you change the view to mimic the original:
- the plate should fill the whole page
- the standards lot number and the upload button should be next to the plate number

Original:
![Screenshot 2024-01-09 at 15 04 05](https://github.com/sanger/quanthub/assets/519327/aac4757e-37d6-468c-bf23-ce64682553a4)

Current:
![Screenshot 2024-01-09 at 15 04 47](https://github.com/sanger/quanthub/assets/519327/08987080-76ad-40fb-b4da-51619fd38cd3)

Reply:
Done